docs/getting-started/integrate/browser.mdx

```diff
@@ -62,7 +62,7 @@ In order for our mock definition to excute during the runtime, it needs to be im
   production. Doing so may lead to a distorted experience for your users.
 </Hint>
 <Action>
-  Import the <code>mocks.js</code> file conditionally following one of the
+  Import the <code>browser.js</code> file conditionally following one of the
```

Comment:
Nice catch! Could you please reference the full path to the module here, like `src/mocks/browser.js`?

Reply:
@kettanaito  I just changed that